packages/form-js-playground/src/Playground.css

```diff
@@ -1,73 +1,158 @@
-.app-root {
+.fjs-pgl-root {
```

Comment:
Out of curiosity: what does `pgl` mean? I first thought it stands for "playground", but then it would rather name `.fjs-plg-*`.

Reply:
Good catch. Needs fix.